src/less/index.less

```diff
@@ -0,0 +1,17 @@
+// Clever Styles
+// Version 0.6.2
+
+// Color and font variables
+@import "./colors.less";
+@import "./fonts.less";
+@import "./type-size.less";
+@import "./grid.less";
+
+// Component Modules
+@import "../Button/Button";
+@import "../ConfirmationButton/ConfirmationButton";
+@import "../SegmentedControl/SegmentedControl";
+@import "../TextInput/TextInput";
+@import "../CopyableInput/CopyableInput";
+@import "../Modal/Modal";
```

Comment:
Ok I updated the new component script. I think I'd rather be explicit about the imports so it's not too magical and we can control the order of the imports.